docs/pages/faq.mdx

```diff
@@ -133,58 +127,22 @@ It can be used simultaneously with React Native Community. Regardless of which C
 
-<BoxLink
-  title="EAS Build"
-  description="EAS Build is designed to work for any React Native project, whether or not you also use Expo Prebuild or manage your own native files. In some situations, you may need to change your project configuration, or it may be incompatible with your app. Learn more."
-  href="/build-reference/limitations"
-  Icon={BuildIcon}
-/>
-
-<BoxLink
-  title="EAS Metadata"
-  description="EAS Metadata helps create or maintain an app in the stores as easily as possible by using one configuration file. In some situations, not be the right fit for a project. Learn more."
-  href="/eas/metadata/faq/#anti-pitch"
-  Icon={EasMetadataIcon}
-/>
-
-<BoxLink
-  title="Notifications"
-  description="Push Notifications are an important feature, no matter what kind of app you're building. In some situations, you might want to know about common issues when setting up push notifications with Expo. Learn more."
-  href="/push-notifications/faq"
-  Icon={NotificationBoxIcon}
-/>
-
-#### EAS and bare React Native projects
-
-EAS Build is compatible with bare (existing) React Native projects (where native directories are checked in version control). When these directories are present, it does not run prebuild, as that could overwrite any manual customizations you have made to the native project files. You'll have to configure the native directories on your own with native tools such as Android Studio or Xcode.
-
-### Expo Go
-
-[Expo Go](/get-started/set-up-your-environment/?redirected=#how-would-you-like-to-develop) provides a quick way to get started with your app development. It comes with a pre-configured set of libraries known as the Expo SDK. This makes experimentation much faster and brings the mobile development experience much closer to the web development experience.
+**We strongly recommend any projects that require additional libraries with native code to migrate to [development builds](/develop/development-builds/introduction/). It's like creating a version of Expo Go that is specifically customized to your app's needs.**
 
-Like any other tool, it too has its limitations:
+## Is Expo eject deprecated?
 
-- Using a third-party library that requires native code
-- Using a third-party push notification service
+Yes, Expo eject is a deprecated term and is no longer necessary. When Expo was first released, apps had larger native binary sizes and didn't support custom native code without "ejecting". This changed in December 2020 with the release of [EAS Build](/build/introduction) which supports any React Native app. The concept of "ejecting" was replaced by the [`npx expo prebuild`](/workflow/prebuild) command in SDK 41 (April 2021), which continuously generates native projects based on the libraries in your project and the app config (**app.json**). The `expo eject` command was fully deprecated in SDK 46 (August 2022).
```

Comment:
```suggestion
Yes, eject is a deprecated term and is no longer necessary. When Expo was first released, apps had larger native binary sizes and didn't support custom native code without "ejecting". This changed in December 2020 with the release of [EAS Build](/build/introduction) which supports any React Native app. The concept of "ejecting" was replaced by the [`npx expo prebuild`](/workflow/prebuild) command in SDK 41 (April 2021), which continuously generates native projects based on the libraries in your project and the app config (**app.json**). The `expo eject` command was fully deprecated in SDK 46 (August 2022).
```
I haven't really heard it called "Expo eject" in terms of a brand name (yes in terms of `expo eject`). It's often just called "ejecting"